blocks-config/uagb-controls/block-icons.js

```diff
@@ -1146,6 +1146,38 @@ const UAGB_Block_Icons = {
 			strokeLinejoin: 'round',
 		} )
 	),
+	loop_builder: el(
+		'svg',
+		{ width: 24, height: 24, viewBox: '0 0 25 24', fill: 'none', className: 'uagb-editor-icons' },
+		el(
+			'g',
+			{ clipPath: 'url(#clip0_2148_7756)' },
```

Comment:
Since we've removed the clipPath tag, can we remove the clipPath attribute from this group tag as well?

```suggestion
			{},
```

Reply:
Should we keep {} or remove it.

Reply:
We'll have to keep it as an empty object or `null`, as [CreateElement](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-element/#createelement) expects the second parameter to be an object.

Reply:
keeping null